### PR TITLE
integrates backend v2

### PR DESCRIPTION
**Pull request by Yelhedhli**

### CODE REVIEW

File: src/pages/UserSessionRestaurants/RestaurantSession.tsx [hunk 1]
```diff
@@ -38,14 +38,18 @@ export const UserSessionRestaurants = () => {
                 console.log(JSON.stringify(event, null, 2))
                 const data = JSON.parse(event.data) 
                 console.log('Event State: ' + data.State)
-                if (data.State == 'CurrRating') {
+                if(data.MessageType == "stateEvent"){
```
Comment (eriklungulescu) [inline src/pages/UserSessionRestaurants/RestaurantSession.tsx:41]: fix spacing

File: src/pages/UserSessionRestaurants/RestaurantSession.tsx [hunk 2]
```diff
@@ -38,14 +38,18 @@ export const UserSessionRestaurants = () => {
...
+                    console.log('stateEvent')
+                    if (data.State == 'CurrRating') {
+                        send('START_RATING')
+                    }
+        
+                    if (data.State == 'Results') {
+                        send('SEE_RESULTS')
+                    }
+                }else if(data.MessageType == "dataEvent"){
```
Comment (eriklungulescu) [inline src/pages/UserSessionRestaurants/RestaurantSession.tsx:50]: fix spacing

File: src/pages/UserSessionRestaurants/RestaurantSession.tsx [hunk 3]
```diff
@@ -38,14 +38,18 @@ export const UserSessionRestaurants = () => {
                 console.log(JSON.stringify(event, null, 2))
                 const data = JSON.parse(event.data) 
                 console.log('Event State: ' + data.State)
-                if (data.State == 'CurrRating') {
+                if(data.MessageType == "stateEvent"){
+                    console.log('stateEvent')
+                    if (data.State == 'CurrRating') {
```
Comment (eriklungulescu) [inline src/pages/UserSessionRestaurants/RestaurantSession.tsx:43]: inconsistent spacing

File: src/pages/UserSessionRestaurants/RestaurantSession.tsx (hunk 2 above)
Comment (Yelhedhli) [reply src/pages/UserSessionRestaurants/RestaurantSession.tsx:50]: get a linter

Comment (eriklungulescu) [inline src/pages/UserSessionRestaurants/RestaurantSession.tsx:50]: consider fixing spacing

**Review by eriklungulescu — APPROVED** [review]
lgtm

**Pull request merged**